app/design/frontend/Magento/storefront/etc/view.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<view xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Config/etc/view.xsd">
+    <media>
```

Comment:
This file is required only if the current theme doesn't have a parent. Also, you only need to override items you need to change, for example:

```
<view xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Config/etc/view.xsd">
    <media>
           <images module="Magento_Catalog">
                <image id="bundled_product_customization_page" type="thumbnail">
                     <width>140</width>
                     <height>140</height>
                </image>
            </images>
     </media>
</view>
```

You will only change `bundled_product_customization_page` image sizes, everything else stays in the core file